webapp/src/projects.tsx

```diff
@@ -568,6 +568,7 @@ export class ProjectsDetail extends data.Component<ProjectsDetailProps, Projects
             clickLabel = lf("Start Tutorial");
         }
         else if (cardType == "codeExample" || cardType == "example") clickLabel = lf("Open Example");
+        else if (cardType == "forumLink") clickLabel = lf("Open in Forum");
```

Comment:
See in forum. @ganicke what is the right word here?

Reply:
I would say either "Open in forum" or "Go to forum" would work